resources/views/hardware/checkin.blade.php

```diff
@@ -113,17 +113,19 @@
                                             </div>
                                         </div>
 
-                                        <!-- Note -->
-                                        <div class="form-group {{ $errors->has('note') ? 'error' : '' }}">
-                                            <label for="note" class="col-sm-3 control-label">
-                                                {{ trans('general.notes') }}
-                                            </label>
-                                            <div class="col-md-8">
-                                                <textarea class="col-md-6 form-control" id="note"
-                                                          name="note">{{ old('note', $asset->note) }}</textarea>
-                                                {!! $errors->first('note', '<span class="alert-msg" aria-hidden="true"><i class="fas fa-times" aria-hidden="true"></i> :message</span>') !!}
-                                            </div>
-                                        </div>
+                        <!-- Note -->
+                        <div class="form-group {{ $errors->has('note') ? 'error' : '' }}">
+                            <label for="note" class="col-md-3 control-label">
+                                {{ trans('general.notes') }}
+                            </label>
+                            @if($snipeSettings->require_checkinout_notes=="1")
+                            <div class="col-md-8 required">
```

Comment:
We can remove this 'required' class here now, due to recent changes.

Reply:
removed